src/AutoRest.CSharp/Common/Output/PostProcessing/PostProcessor.cs

```diff
@@ -90,7 +90,7 @@ protected async Task<TypeSymbols> GetTypeSymbolsAsync(Compilation compilation, P
             documentCache.ToDictionary(kv => kv.Key, kv => kv.Value.ToImmutableHashSet<INamedTypeSymbol>(SymbolEqualityComparer.Default)));
     }
 
-    protected virtual bool ShouldIncludeDocument(Document document) => !GeneratedCodeWorkspace.IsSharedDocument(document);
+    protected virtual bool ShouldIncludeDocument(Document document) => !GeneratedCodeWorkspace.IsSharedDocument(document) && !GeneratedCodeWorkspace.IsGeneratedTestDocument(document);
```

Comment:
`IsGeneratedTestDocument` do you mean the xml doc which contains sample code? If so, better to call `IsGenereratedSampleDocument`. In future, we will generate Test,  incase confusing.

Reply:
No, this refers to anything generated to the tests directory.
We are generating source code (into src directory) and test code (into tests directory). Samples are part of it.